AdornUI-documentation/navigation.html

```diff
@@ -0,0 +1,80 @@
+    <meta name="keywords" content="HTML, CSS, JS" />
+    <title>AdornUI-Navigation</title>
+    <link rel="icon" type="image/x-icon" href="./images/adornui-logo.png">
+    <title>AdornUI</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="../AdornUI-Stylesheets/AdornUi.css" />
+</head>
+
+<body>
+    <div class="page-layout">
+        <header class="page-header">
+            <h1 class="lib-name-1">Adorn <span class="lib-name-2">UI</span></h1>
+            <ul class="nav-items-wrapper">
+                <li class="nav-item"><a class="nav-item-link" href="home.html">Home</a></li>
+                <li class="nav-item"><a class="nav-item-link" href="alert.html">Documentation</a></li>
+            </ul>
+        </header>
+        <aside class="sidebar">
+            <h2 class="sidebar-heading">Components</h2><span><i class="fa fa-angle-down"></i></span>
+            <div class="sidebar-items">
+                <a class="sidebar-item-link" href="alert.html">Alerts</a>
+                <a class="sidebar-item-link" href="badges.html">Badges</a>
+                <a class="sidebar-item-link" href="avatar.html">Avatar</a>
+                <a class="sidebar-item-link" href="card.html">Cards</a>
+                <a class="sidebar-item-link" href="button.html">Buttons</a>
+                <a class="sidebar-item-link" href="input.html">Input</a>
+                <a class="sidebar-item-link" href="navigation.html">Navigation</a>
+            </div>
+        </aside>
+        <main>
+            <section class="component-doc-container">
+                <h2 class="component-title">Navigation</h2>
+                <h5 class="component-subtitle">Standard Navigation</h5>
+                <h5 class="component-desc">Navigation Component is used for navigating between web pages.It includes links for the other sections on the website. This navigation component is responsive and has a hamburger menu.</h5>
+                <div class="component-container">
+                    <div class="navigation-container flex-hz">
+                        <div class="navigation-item-1">
+                            <h5 class="lib-name-1">Adorn <span class="lib-name-2">UI</span></h5>
+                        </div>
+                        <i class="fa fa-bars"></i>
+                        <div class="navigation-item-2">
+                            <label class="search-bar">
+                                <span><i class="fa fa-search"></i></span>
+                                <input type="text" class="search-input" placeholder="search.." />
+                            </label>
+                        </div>
+                        <div class="navigation-item-3 flex-hz">
+                            <button class="btn btn-primary">Login</button>
+                            <i class="fa fa-shopping-cart"><span class="badge-w-icon">1</span></i>
```

Comment:
As fa is generic class name for all icons. It's better to include separate class while applying any CSS to icons